model_analyzer/entrypoint.py

```diff
@@ -345,6 +346,27 @@ def create_output_model_repository(config):
             os.mkdir(config.output_model_repository_path)
 
 
+def fail_if_server_already_running(client, config):
+    """ 
+    Checks if there is already a Triton server running
+    If there is and the launch mode is not 'remote', throw an exception
+    Else, nothing will happen
+    """
+    if config.triton_launch_mode == 'remote':
+        return
+
+    is_server_running = True
+    try:
+        client.is_server_ready()
+    except:
+        is_server_running = False
+    finally:
+        if is_server_running:
+            raise TritonModelAnalyzerException(
+                "Detected Triton server already running. Model Analyzer will launch a Triton server for you and requires that a server is not already running."
```

Comment:
Small nit: "Another application is already running on the same port. In 'docker' and 'local' mode, Model Analyzer needs to launch an instance of Triton Server and requires that the HTTP/GRPC port is not occupied by another application (including an instance of Triton Server)."